src/pages/en/guides/markdown-content.mdx

```diff
@@ -497,6 +497,12 @@ You can add frontmatter properties to all of your Markdown and MDX files by usin
     }
     ```
 
+    :::tip
+    <Since v="2.0.0" />
```

Comment:
Looks like we need to import this component to fix the build. This at the top of the file should do it:
```js
import Since from '~/components/Since.astro'
```

Reply:
Ah yep, was modifying this file mid-MDX migration and missed it 👍